app/main.py

```diff
@@ -1 +1,48 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self, name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            eaten_points = self.appetite
+            return eaten_points
+        self.appetite = 0
+        return self.appetite
```

Comment:
There is a logical error here. If the animal is not hungry, the method sets `self.appetite` to 0 and returns it. This might not be the intended behavior, as it permanently changes the animal's appetite. Consider returning 0 directly without modifying `self.appetite`.